### ISSUE TITLE
VC2015: Call from 0x7b83c3cc to unimplemented function api-ms-win-crt-runtime-l1-1-0.dll._initialize_onexit_table, aborting 

### ISSUE THREAD

**Issue by slyscorpion** [Winetricks/winetricks#622]
Wine 1.9.6 with WoW 64 installed from the official Arch repo: [https://www.archlinux.org/packages/multilib/x86_64/wine/](url)

Winetricks version: 20160311 - sha1sum: d5e01087e523b20c7770f65dd8490436ae61d02f installed via this official Arch package: [https://www.archlinux.org/packages/community/any/winetricks/](url)

Clean 32-bit prefix which I then installed d3dx9 vcrun 2010 vcrun2012 vcrun2015 (I did all those because those redistributables come with the GOG version of the game.)

It looks like [https://github.com/Winetricks/winetricks/issues/575](url) has popped up again and that is why I am opening up a bug report here. Please let me know if I need to file this bug with WineHQ or not as it's my first time filing an issue with winetricks :) 

I just bought the GOG version of Grim Dawn and I installed the aforementioned winetricks, set the Windows version to XP and attempted to run the game with "wine Grim Dawn.exe". 

Here is the terminal output: 

`wine Grim\ Dawn.exe 
wine: Call from 0x7b83c3cc to unimplemented function api-ms-win-crt-runtime-l1-1-0.dll._initialize_onexit_table, aborting
fixme:vcruntime:__telemetry_main_return_trigger (0x340000)
err:module:attach_process_dlls "MSVCP140.dll" failed to initialize, aborting
err:module:LdrInitializeThunk Main exe initialization for L"C:\\GOG Games\\Grim Dawn\\Grim Dawn.exe" failed, status 80000100
`

I wanted to try and set some overrides for msvcp140 but wineconfig only offers msvcp120 and there is no option to select msvcp140 at all...

Please let me know if you require any further info from me.

**Comment by austin987** [comment]
That terminal output indicates something deeper is wrong.

- Are you sure that is a 32-bit prefix?
- What directory / file system is that WINEPREFIX mounted on?

Can you do:
- `ls -al ~/.wine/drive_c/windows/system32/{api*,msvcp140.dll}`
-` ls -al ~/.wine/drive_c/windows/syswow64/{api*,msvcp140.dll}`

**Comment by slyscorpion** [comment]
Hi Austin, 

Yes, I made sure it was a 32-bit prefix by setting it up with the WINEARCH=win32 setting when creating it and I also did not receive the "You are using a 64-bit prefix" warning from winetricks.

As for the ls commands, I'll need to reinstall the stock version of Wine from the Arch repos as I've got wine-gaming-nine installed from there in order to be able to run Grim Dawn as it requires CSMT. 

I'll do this within a day or so :)

**Comment by qwertychouskie** [comment]
The `ls` commands should work fine with `wine-gaming-nine` installed, as they are listing files in the wineprefix.  Also, ~ is the same thing as `/home/username/`, except it can be copy-pasted without having to insert your own username.

**Comment by slyscorpion** [comment]
Hi, 

I've removed `wine-gaming-nine` and installed `wine-1.9.7` and I've also updated winetricks to 20160329 - sha1sum: 8602b7cd37f520fc37aa1e910d61360c5689d107

I then created a clean 32-bit prefix and then ran "winetricks vcrun2015" and the command worked as expected. 

Here is the output of `ls -al ~/.wine/drive_c/windows/system32/{api*,msvcp140.dll}` http://pastebin.com/1Q5jH2rq after installing vcrun2015 on my 32-bit prefix.

In conclusion, I think this may have been an issue on my side somewhere but I am not sure just where. In any case, sorry for any confusion my bug report may have caused.

**Comment by austin987** [comment]
Glad you got it working.
